test/unit/interfaces/express.ts

```diff
@@ -45,7 +44,7 @@ describe('expressInterface', function() {
           },
         },
         createLogger({logLevel: 4}));
-    stubbedConfig.lacksCredentials = function() {
+    (stubbedConfig as {} as {lacksCredentials: Function}).lacksCredentials = function() {
```

Comment:
Can you use Object.assign to assign stubbedConfig.lacksCredentials a a single statement? We should avoid 2-layer typecasts like these.

Reply:
That's a good idea, but for this PR since it is a PR for translating to TypeScript, I didn't want to change the structure of the code unless I needed to.

Reply:
Sounds good